ngx-fudis/projects/ngx-fudis/src/lib/components/form/error-summary/error-summary.component.spec.ts

```diff
@@ -194,17 +193,16 @@ describe('ErrorSummaryComponent', () => {
   };
 
   describe('Contents', () => {
-    it('helper texts are displayed properly', async () => {
+    it('default title is displayed properly', async () => {
       await wrapperFixture.whenStable().then(() => {
         wrapperFixture.detectChanges();
-        const renderedHelpText = getElement(
+        const renderedTitle = getElement(
           wrapperFixture,
           '.fudis-error-summary__errors fudis-body-text',
         );
 
-        // Hidden icon text + Help Text
-        expect(renderedHelpText.textContent).toBe(
-          'Attention:\u00A0There were errors you need to fix',
+        expect(renderedTitle.textContent).toBe(
```

Comment:
This default value test is great, but there should be also a test for app provided Input prop